dot/parachain/types/statement_test.go

```diff
@@ -1,21 +1,36 @@
 // Copyright 2023 ChainSafe Systems (ON)
 // SPDX-License-Identifier: LGPL-3.0-only
 
-package parachain
+package parachaintypes
 
 import (
+	_ "embed"
 	"errors"
+	"fmt"
 	"math"
 	"testing"
 
-	parachaintypes "github.com/ChainSafe/gossamer/dot/parachain/types"
 	"github.com/ChainSafe/gossamer/lib/common"
 	"github.com/ChainSafe/gossamer/pkg/scale"
 	"github.com/stretchr/testify/require"
+	"gopkg.in/yaml.v3"
 )
 
+//go:embed testdata/statement.yaml
+var testDataStatementRaw string
+
+var testDataStatement map[string]string
+
+func init() {
+	err := yaml.Unmarshal([]byte(testDataStatementRaw), &testDataStatement)
```

Comment:
you can embed the yaml to []byte directly
```suggestion
var testDataStatementRaw []byte

var testDataStatement map[string]string

func init() {
	err := yaml.Unmarshal(testDataStatementRaw, &testDataStatement)
```